ui/src/layouts/chats/mod.rs

```diff
@@ -120,25 +119,10 @@ pub fn ChatLayout(cx: Scope) -> Element {
                                 })
                                 .await
                                 .expect("Should succeed");
-                                if !files_local_path.is_empty() {
-                                    let new_files: Vec<Location> = files_local_path
-                                    .iter()
-                                    .map(|path| Location::Disk { path: path.clone() })
-                                    .collect();
-                                let mut current_files: Vec<_> = state
-                                    .read()
-                                    .get_active_chat()
-                                    .map(|f| f.files_attached_to_send)
-                                    .unwrap_or_default()
-                                    .drain(..)
-                                    .filter(|x| !new_files.contains(x))
-                                    .collect();
-                                    current_files.extend(new_files);
-                                    let active_chat_id = state.read().get_active_chat().map(|f| f.id).unwrap_or(Uuid::nil());
-                                    state
-                                        .write()
-                                        .mutate(Action::SetChatAttachments(active_chat_id, current_files));
-                                }
+                                let active_chat_id = state.read().get_active_chat().map(|f| f.id).unwrap_or(Uuid::nil());
```

Comment:
If I recall, `Uuid` default is `Uuid::nil()` so we could just do an unwrap or default if you will be using the default id.

```suggestion
                                let active_chat_id = state.read().get_active_chat().map(|f| f.id).unwrap_or_default();
```